jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+</aside>
+
+### Example Template
+
+When you want to author an orb, you may wish to use this example template to quickly and easily create a new orb with all of the required concepts. This example includes each of the three top-level concepts of orbs. While any orb can be equally expressed as an inline orb definition, it will generally be simpler to iterate on an inline orb and use ```circleci config process .circleci/config.yml``` to check whether your orb usage matches your expectation.
+```
+version: 2.1
+orbs:
+  inline_example:
+    jobs:
+      my_inline_job:
+        parameters:
+          greeting_name:
+            type: string
+            default: olleh
+        executor: my_inline_executor
+        steps:
+          - my_inline_command:
+              name: <<parameters.greeting_name>>
+    commands:
+      my_inline_command:
+        parameters:
+          name:
+            type: string
+        steps:
+          - run: echo "hello <<parameters.name>>, from the inline command"
+    executors:
+      my_inline_executor:
+        parameters:
+          version:
+            type: string
+            default: "2.4"
+        docker:
+          - image: circleci/ruby:<<parameters.version>>
+
+workflows:
+  build-test-deploy:
+    jobs:
+      - inline_example/my_inline_job:
+          name: build
+      - inline_example/my_inline_job:
+          name: build2
+          greeting_name: world
+```
+
+## Publishing an Orb
+
+This section covers the tooling and flow of authoring and publishing your own orbs to the orb registry.
+
+Orbs may be authored inline in your config.yml file or authored separately and then published to to the orb registry for reuse across projects.
+
+[WARNING] Orbs are always world-readable. All published orbs can be read and used by anyone. They are not limited to just the members of your organization. In general, CircleCI strongly recommends that you do not put secrets or other sensitive variables into your configuration. Instead, use contexts or project environment variables and reference the names of those environment variables in your orbs.
+
+### Prerequisites
+
+Before publishing an orb, you will need to first opt-in to the new 3rd-Party Software terms and turn on orb publishing for your organization.
+
+<aside class="notice">
+Only an organization administrator can opt-in to the 3rd party Software terms. The organization admin will need to navigate to the organization Settings tab and complete the form on the Security page.
+</aside>
+
+### Orb Publishing Concepts
+
+The sections below describe important concepts that you should understand before publishing an orb.
+
+#### Orb Registry
+
+Each installation of CircleCI has a single orb registry. The registry on circleci.com serves as the master source for all certified namespaces and orbs and is the only orb registry that users of circleci.com can use.
```

Comment:
I think it's worth clarifying something like "Each Server installation will have its own registry" (but like, with better words).

Otherwise, for folks coming from `.com` the distinction between `installation` and `registry` could get a bit murky. If you've only ever used `.com`, then the presence of a Server product will not be the first thing to come to mind with the word "installation".

Reply:
agreed, changed this to CircleCI has a single orb registry. The registry on `circleci.com` serves as the master source for all certified namespaces and orbs. This master source is the only orb registry available to users of `circleci.com`.

<aside class="notice">
Orbs are not yet supported for CircleCI installed on your private servers or cloud.